manifests/params.pp

```diff
@@ -37,7 +37,7 @@
   $db_username = 'foreman'
   # Generate and cache the password on the master once
   # In multi-puppetmaster setups, the user should specify their own
-  $db_password = extlib::cache_data('foreman_cache_data', 'db_password', extlib::random_password(32))
+  $db_password = Sensitive(extlib::cache_data('foreman_cache_data', 'db_password', extlib::random_password(32)))
```

Comment:
I am worried that our installer doesn't support Sensitive yet. That would be a blocker to merging this. We are entering our stablization period next week.

I feel like we can find a compromise if we don't default to using Sensitive for now but do merge the other changes to accept `Sensitive`. Do you agree or would you prefer to wait until after we've had a chance to check out installer?

Reply:
I removed the `Sensitive` here.

Reply:
To be clear, I was talking about all changes in params.pp, not just this single one.

Reply:
👍  I removed all `Sensitive` from `params.pp` and left a `TODO`-Reminder.